src/terrain/routes.clj

```diff
@@ -3,7 +3,7 @@
         [clojure-commons.lcase-params :only [wrap-lcase-params]]
         [clojure-commons.query-params :only [wrap-query-params]]
         [compojure.api.middleware :only [wrap-exceptions]]
-        [compojure.core]
+        [compojure.api.sweet]
```

Comment:
I couldn't get it to work with the transitive dependency. I haven't tried to figure out why yet.

Reply:
It appears that all of our other services that use compojure-api have direct dependencies on compojure-api as well.

Reply:
I thought maybe that was only because they hadn't been fully migrated to `common-swagger-api` yet, but maybe there's something I'm missing and it's not possible 🤷‍♂️

Reply:
It definitely seems like the transitive dependency should work. I'm not sure why it isn't working at this point. 🤷‍♀️